tutorials/24_Building_Chat_Application.ipynb

```diff
@@ -0,0 +1,1953 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TEebQqubxa4G"
+   },
+   "source": [
+    "# Tutorial: Building a Human-like Chat Application\n",
+    "\n",
+    "- **Level**: Intermediate\n",
+    "- **Time to complete**: 10 minutes\n",
+    "- **Nodes Used**: `PromptNode`, `ConversationalAgent` and `ConversationSummaryMemory`\n",
+    "- **Goal**: After completing this tutorial, you will have learned how to use ConversationalAgent to build a conversational chat application\n",
+    "- **Prerequisites**: A [Hugging Face API Key](https://huggingface.co/settings/tokens)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "DQLWWW1Yy_Ta"
+   },
+   "source": [
+    "## Overview\n",
+    "\n",
+    "A [ConversationalAgent]() is a type of agent that is specifically implemented to create chat applications. With its memory integration, the new ConversationalAgent enables human-like conversation with large language models (LLMs).  \n",
+    "\n",
+    "This tutorial introduces you to the ConversationalAgent, ConversationSummaryMemory and explains how you can create your conversational chat application.  \n"
```

Comment:
`ConversationalAgent` and `ConversationSummaryMemory`

Reply:
@TuanaCelik I usually don't mark component names as `coded`. Do you think I should change it in the rest tutorial as well?

Reply:
I tend to not, but usually try to for when I'm first introducing I do. I think it's your style choice :) Feel free to ignore comment